charts/polkadot-registrar-challenger/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+description: Polkadot Challenger
```

Comment:
What is this file for?

Reply:
This is called Helm Chart. It is a way to package, configure, and deploy a set of kubernetes resource: https://helm.sh/